app/animation_manager.py

```diff
@@ -48,18 +58,27 @@ def update(data, rects, iteration):
         plt.close()
 
     def visualize_quick_sort(self):
+        '''
+            Description: Visualization of the quick sort algorithm
+        '''
         data = self._data.copy()
         generator = self._sorting_algorithms.quicksort(data, 0, len(data)-1, self._k_digits)
 
         fig, ax = plt.subplots()
-        ax.set_title(f"Quick Sort O(n) with k = {self._k_digits}")
+        ax.set_title(f"Quick Sort O(n) with k = {self._k_digits}")  #set title for the graph
         bar_sub = ax.bar(range(len(self._data)), self._data, align="edge")
 
         ax.set_xlim(0, self._data_size)
         text = ax.text(0.02, 0.95, "", transform=ax.transAxes)
         iteration = [0]
 
         def update(data, rects, iteration):
+            '''
```

Comment:
- in description also say its a helper function that is used in the animation method to update the graph.
- the params data is actually not just a list but a dictionary with the list and other variables. Please include this in the comments